pallets/gear/src/runtime_api.rs

```diff
@@ -302,7 +302,7 @@ where
             None,
             None,
             payload,
-            BlockGasLimitOf::<T>::get() / 4,
+            BlockGasLimitOf::<T>::get() * 6,
```

Comment:
If your are using * 6 for each, then this must be separate constant with explanation why * 6.